CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Enhancements
 
 - Cache parsed DSN ([#2365](https://github.com/getsentry/sentry-dart/pull/2365))
+- Switching from traditional screenshot to view hierarchy for screenshots which allows redacting ([#2361](https://github.com/getsentry/sentry-dart/pull/2361))
```

Comment:
The changelog entry should be under `## Features`, if there is none please create it